src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -96,13 +94,16 @@ class JSONSerializer(Serializer):
 
     See `Subject name strategy <https://docs.confluent.io/current/schema-registry/serializer-formatter.html#subject-name-strategy>`_ for additional details.
 
-    Note:
-        The ``title`` annotation, referred to as a record name
-        elsewhere in this document, is not strictly required by the JSON Schema
-        specification. It is however required by this Serializer. This
-        annotation(record name) is used to register the Schema with the Schema
-        Registry. See documentation below for additional details on Subjects
-        and schema registration.
+    Notes:
+        The ``title`` annotation, referred to elsewhere as a record name
+        is not strictly required by the JSON Schema specification. It is
+        however required by this serializer in order to register the schema
+        with Confluent Schema Registry.
+
+        Prior to serialization, all values must first be converted to
```

Comment:
What are these "values"?
It could be the message value, but then how do I serialize the key?
Or it could be a dict value? As a user I wouldn't know.

Since everything is an object in Python, and every object can (potentially) be serialized, I think we should stick with `object` being the input to a serializer (and output from a deserializer), and not use other ambiguous terms.

Reply:
yes, good plan.